Microsoft.Toolkit.Uwp/Helpers/DispatcherHelper.cs

```diff
@@ -204,17 +188,35 @@ public static Task<T> AwaitableRunAsync<T>(this CoreDispatcher dispatcher, Func<
         /// <returns>Awaitable Task</returns>
         public static Task AwaitableRunAsync(this CoreDispatcher dispatcher, Func<Task> function, CoreDispatcherPriority priority = CoreDispatcherPriority.Normal)
         {
-            return dispatcher.AwaitableRunAsync<object>(
```

Comment:
@skendrot complained about unneccessary awaits in the older PR, and I think he is right. Here we can save one more await...